docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -87,6 +87,10 @@ Webhooks can be used to receive data from Saleor when particular events happen.
 | Name                           | Description                                                                                                                                                                                                             |
 | -------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `ANY_EVENTS`                   | Receive webhooks when any of the available events is triggered (wildcard event).                                                                                                                                        |
+| `APP_CREATED`                  | A new app created.                                                                                                                                                                                                      |
```

Comment:
Are apps created or installed?

Reply:
we send this event when an app is created and after `successful installation/reinstallation`

Reply:
I'd rather emit `APP_INSTALLED` for both cases.

Reply:
done, PR with change [here.](https://github.com/saleor/saleor/pull/9757)